File: src/main/java/com/raoulvdberge/refinedstorage/screen/EmptyScreenInfoProvider.java
package com.raoulvdberge.refinedstorage.screen;

public class EmptyScreenInfoProvider implements IScreenInfoProvider {
    @Override
    public int getVisibleRows() {
        return 3;
    }

    @Override
    public int getRows() {
        return 0;
    }

    @Override
    public int getCurrentOffset() {
        return 0;
    }

    @Override
    public String getSearchFieldText() {
        return "";
    }

    @Override
    public int getTopHeight() {
        return 0;
    }

    @Override
    public int getBottomHeight() {
        return 0;
    }

    @Override
    public int getYPlayerInventory() {
        return 0;
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/screen/EmptyScreenInfoProvider.kt
package com.refinedmods.refinedstorage.screen

class EmptyScreenInfoProvider : IScreenInfoProvider {
    override fun getVisibleRows(): Int {
        return 3
    }

    override fun getRows(): Int {
        return 0
    }

    override fun getCurrentOffset(): Int {
        return 0
    }

    override fun getSearchFieldText(): String? {
        return ""
    }

    override fun getTopHeight(): Int {
        return 0
    }

    override fun getBottomHeight(): Int {
        return 0
    }

    override fun getYPlayerInventory(): Int {
        return 0
    }
}